### PR TITLE
Stefan Clem

### PR DESCRIPTION
**Pull request by iepoch**

### CODE REVIEW

File: baseball/src/Components/Display/Display.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Dashboard from "../Dashboard/Dashboard";
+
```
Comment (juliankohlman) [inline baseball/src/Components/Display/Display.js:3]: Nice job composing your components, and making the `dashboard` a stateless 'presenter' component Stefan 🔥

File: baseball/src/Components/Dashboard/Dashboard.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    const ballBtn = components.getByTestId("ball-btn");
+    const strikeBtn = components.getByTestId("strike-btn");
+    const foulBtn = components.getByTestId("foul-btn");
+    const hitBtn = components.getByTestId("hit-btn");
+
+    fireEvent.click(ballBtn);
+    fireEvent.click(strikeBtn);
+    fireEvent.click(foulBtn);
+    fireEvent.click(hitBtn);
+
```
Comment (juliankohlman) [inline baseball/src/Components/Dashboard/Dashboard.test.js:23]: This looks awesome Stefan, super easy to read and understand what's going on within each test 👍

File: baseball/src/Components/Dashboard/Dashboard.test.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    expect(ballBtn).toHaveTextContent(/ball/i);
+    expect(strikeBtn).toHaveTextContent(/strike/i);
```
Comment (juliankohlman) [inline baseball/src/Components/Dashboard/Dashboard.test.js:25]: A nice and efficient way to check that those buttons have a default value 👍

File: baseball/src/Components/Display/Display.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        strikes: 0
+    });
+    } else {
+    this.setState({
+        ...this.state,
+        balls: this.state.balls + 1
+    });
+    }
+};
+
```
Comment (juliankohlman) [inline baseball/src/Components/Display/Display.js:24]: I think that based on the assignment and requirements, having separate handlers as you have is the best way to go 👍 Looking at your code made me think 🤔 of creating a sort of generic `pitchHandler` and then having a random `pitch event` generator to pass to that handler for a potential ball, strike, foul, or hit. Good stuff Stefan 🔥

Comment (iepoch) [reply baseball/src/Components/Display/Display.js:24]: That would be cool Julian. I like to see how you randomize the pitchEvents.

**Pull request merged**